lib/Runtime/Base/PropertyRecord.h

```diff
@@ -112,7 +112,7 @@ namespace Js
     struct BuiltInPropertyRecord
     {
         PropertyRecord propertyRecord;
-        char16 buffer[LEN];
+        WCHAR buffer[LEN];
```

Comment:
My understanding was that we were trying to get aware from WCHAR (along with other all-uppercased types)?